aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/diamond_clicker.exp

```diff
@@ -1,76 +1,62 @@
 processed 4 tasks
 
 task 1 'print-bytecode'. lines 4-35:
-Error: extended checks failed:
+// Move bytecode v7
```

Comment:
Could you provide a brief comment about this change in output due to compiler v2? I see that this change pre-exists (as seen from .v2_exp).

Reply:
yes, we have a V1-only bug for this: https://github.com/aptos-labs/aptos-core/issues/14722